posts/azure-seen-by-developers.md

```diff
@@ -0,0 +1,190 @@
+
+In the left menu there are useful shortcuts such as "Configuration" which allows you to configure the environment variables of the web app or "Advanced tools" which allows you to access the application logs of the web app.
+
+![Figure 6 : "Configuration" view of a web app](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_6.png)
+
+This view lists the environment variables that the web app retrieves depending on the environment in which it is hosted. Thus, it is also possible to edit them but also to create or delete them.
+
+![Figure 7 : "Advanced tools" view of a web app](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_7.png)
+
+This very simple view provides access to _Kudu_, Azure's web app back office that stores the web app's application logs.
+
+![Figure 8 : "LogFiles" view of Apache Kudu](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_8.png)
+
+This view is the _LogFiles_ directory of the server where the web app is hosted. It contains the application logs generated by the web app thanks to the integration of _Application Insights_ in the application.
+
+Note that it is possible to read the logs directly from the browser or to download them locally on your machine.
+
+![Figure 9 : Application Insights online documentation](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_9.png)
+
+To configure Application Insights (Azure Monitor feature) in your application, you can follow [this documentation](https://docs.microsoft.com/fr-fr/azure/azure-monitor/app/app-insights-overview) which redirects to the web page shown in Figure 9.
+
+## Databases	
+
+![Figure 10 : View of a SQL Server DB](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_10.png)
+
+This view is the home page of a SQL Server DB. Among other things, it allows you to access the SQL Server via the _srvint_ link located on the right side of the screen. It also provides two monitoring indicators on the DB which are the CPU usage and the disk fill rate.
+
+In the left menu there are useful shortcuts such as "Query Performance Insight" which informs about the performance of the database by type of metrics and "Connection Strings" which lists the connection strings to the DB.
+
+![Figure 11 : Query Performance Insight page view](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_11.png)
+
+This list view allows you to visualize the performance of your DB with several very useful filters. Indeed, you can choose to observe the CPU performance, but also the input/output in terms of data or logs, the duration of the queries and finally the number of execution of the queries, all on time intervals chosen by another filter. Thus, this tool can be very useful when the need to analyze or improve the performance of the DB is felt.
```

Comment:
This list view allows you to visualize the performance of your DB with several useful filters. You can choose to observe the CPU performance, the input/output in terms of data or logs, the duration of the queries, and the number of executions of the queries -- all on-time intervals chosen by another filter. Therefore, this tool can be useful when there is a need to analyze or improve the performance of the DB.